app/api/leaderboard/functions.ts

```diff
@@ -39,13 +39,15 @@ export const getLeaderboardData = async (
 
   const contributors = await getContributors();
 
-  const data = contributors
+  let data = contributors
```

Comment:
Let's try to keep it immutable as much as possible

```suggestion
  const data = contributors
```